modules/ROOT/pages/distributed-file-polling.adoc

```diff
@@ -0,0 +1,23 @@
+= Distributed File Polling
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: file polling, cluster
+
+*_Enterprise Edition_*
+
+Some connectors, such as the File connector or FTP connector, regularly poll directories and read certain files as they are created in the directories polled. These files may reside on a remote file system, including file systems of nodes belonging to a Mule High Availability (HA) Cluster.
+
+In Mule 4, distributed file polling makes it possible to poll files in all cluster nodes. Enabled by default, this feature is used by the following connectors:
+
+* xref:connectors::file/file-connector.adoc[File Connector]
+* xref:connectors::ftp/ftp-connector.adoc[FTP Connector]
+* xref:connectors::sftp/sftp-connector.adoc[SFTP Connector]
+
+You can configure connectors to only poll from the primary node, `@PrimaryNodeOnly`. Therefore, ignoring the default setting set by the Mule runtime engine. This is only available in Mule 4.x
```

Comment:
Suggested fix to fix sentence (which is a sentence fragment):  "You can configure ... `@PrimaryNodeOnly`, which ignores the default ..."

"This" needs to modify something. Perhaps "This feature is only ..."  That said, if these are Mule 4 docs, it might not be necessary to mention that it's only available in Mule 4. Your call. If you think it will help 3.x customers who are moving to Mule 4, leave it in.

Reply:
Updated.